core/trino-main/src/test/java/io/trino/sql/query/TestColumnMask.java

```diff
@@ -180,7 +180,7 @@ public void testSimpleMask()
     }
 
     @Test
-    public void testMultipleMasksOnSameColumn()
```

Comment:
Removing this test, since it's not allowed to have more than one masking on the same column name.
https://github.com/trinodb/trino/blob/35986d7024380f8c4d99a57bc6187e95199bdb10/lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/TableAccessControlRule.java#L72